test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -636,3 +636,99 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         $result.Error | Should BeNullOrEmpty
     }
 }
+
+Describe "Bug fixes" -Tags "Feature" {
+
+    Context "InFile parameter negative tests" {
+
+        function ValidateWebError
+        {
+            param ($testCase)
+
+            It "$($testCase.Name)" {
+
+                try
+                {
+                    $sb = [scriptblock]::Create($testCase.Command)
+                    & $sb
+                    throw "CodeExecuted"
+                }
+                catch
+                {
+                    $_.FullyQualifiedErrorId | should be $testCase.ExpectedFullyQualifiedErrorId
+                }
+            }
+        }
+
+        $testCases = @(
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile ""'
+                Command = 'Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile ""'
+                ExpectedFullyQualifiedErrorId = 'WebCmdletInFileNotFilePathException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile'
+                Command = 'Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile'
+                ExpectedFullyQualifiedErrorId = 'MissingArgument,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile c:\pathThatDoesNotExist\foo.txt'
+                Command = 'Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile c:\pathThatDoesNotExist\foo.txt'
```

Comment:
> -InFile c:\pathThatDoesNotExist\foo.txt

This test will fail on unix plat because it will throw an error with FQID to be `DriveNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestCommand`